src/Core/System/Window.cpp

```diff
@@ -202,6 +210,10 @@ namespace obe::system
         m_window.setKeyRepeatEnabled(false);
 
         this->apply_view();
+        events::Window::Size new_size { m_width, m_height };
+        events::Window::Size previous_size { 0, 0 };
+        e_window->trigger(
```

Comment:
Not sure we should trigger `Resized` here, eventually, we could add a `Created` or `Opened` event for this